api/router.go

```diff
@@ -20,6 +21,6 @@ func ConfigGinRouter(router *gin.Engine) {
 	configProgressRouter(router)
 	configServerNotificationsRoutes(router)
 	configTokenRouter(router)
-	configWorkerRouter(router)
+	configWorkerRouter(router, dao.NewWorkerDao())
```

Comment:
what do we do if a router needs multiple daos? pass them all here? Create a struct in the dao package that contains all sub-daos for the routes? Any idea which might be the best option?

Reply:
I feel like a struct containing all DAOs would be the best option. Each router receives a pointer to that struct object and uses the DAOs as needed. Another benefit of that approach is that if some endpoint ever needs other DAO objects we don't have to add additional parameters.